packages/design-system/src/Link/Link.styles.tsx

```diff
@@ -4,6 +4,21 @@ import { Icon } from "../Icon";
 import { LinkKind } from "./Link.types";
 
 const Variables = css`
+  --ads-v2-action-link-primary-label-default-fg: var(--ads-v2-color-fg-brand);
```

Comment:
Where are these variables being used?

Reply:
Link and Tag is not being changed until your PR goes to release